p5/sketch/userspace.py

```diff
@@ -210,7 +210,11 @@ def exit(*args, **kwargs):
     :param kwargs: keyword-arguments to pass to Python's builtin
         `exit()` function.
     """
-    default_sketch.show(visible=False)
+    try:
```

Comment:
I would avoid a try catch.. just in the odd chance that something else throws an `AttributeError`. A simple check for `default_sketch == None` should suffice.